README.md

```diff
@@ -217,6 +217,8 @@ DYNDNS:
 
 Google:
 
+Note that you must enable Dynamic DNS in the Synthetic records section of DNS management. The username and password is generated once you . You cannot create a wildcard entry for Dynamic DNS but you can CNAME `"*"` to the `"@"` domain for the same effect.

```

Comment:
> You cannot create a wildcard entry for Dynamic DNS but you can CNAME `"*"` to the `"@"` domain for the same effect.

You mean that setting a CNAME record with key `"*"` and value `"@"` has the same effect right?

Reply:
Great catch...Yes, same effect.  I updated my text in the previous comment to help clarify.